src/test/java/com/cdancy/bitbucket/rest/features/CommentsApiLiveTest.java

```diff
@@ -81,7 +81,7 @@ public void init() {
         
         final String randomChars = TestUtilities.randomString();
         final ProjectKey proj = ProjectKey.create(project);
-        final MinimalRepository repository = MinimalRepository.create(repo, null, proj);
+        final MinimalRepository repository = MinimalRepository.create(42, repo, null, proj);
```

Comment:
The big thing is: do the comments live tests still pass and work as expected though we've changed this POJO? I think it might be better to keep the original `create` method and just provide an overloaded one for the new use-case which is allowed. Or maybe I'm over thinking things...

Reply:
I routinely ran the live and mock tests while working on this and nothing failed. I'll look into overloading in any case

Reply:
I'm in the process of reverting this change after having simplified the UpdateTask properties

Reply:
+1